platforms/android/demo/src/main/java/com/mapzen/tangram/android/MainActivity.java

```diff
@@ -237,5 +241,14 @@ public void run() {
             }
         });
     }
+
+    @Override
+    public void onSceneUpdateError(List<SceneUpdateStatus> updateStatuses) {
+        for (SceneUpdateStatus status : updateStatuses) {
+            Log.d("Tangram", "Scene update errors "
```

Comment:
If we have something on those lines:
```java
try {
    map.loadScene("scene.yaml", sceneUpdates);
} catch(SceneUpdateException) {

} 
```
Around anything that applies the scene updates. 

Some problem with that pattern is that some of our API calls that applies the scene updates are not synchronous, so if we catch an error for a scene update and throw an exception, the `try` block would be already out of scope and the exception thrown on a different context (basically our `update` loop).

Another solution, is that we could surround our `update` loop by a `try` `catch` for those exceptions.